jquery.serializejson.js

```diff
@@ -154,13 +159,14 @@
                 }
 
                 if (isArray(val)) {
-                    return $.map(val, function(val) {
-                        return { name: el.name, value: val.replace(rCRLF, "\r\n"), el: el };
-                    } );
+                  if(!val.length) {
+                      return { name: el.name, el: el }
+                  }
```

Comment:
when would the array not have a length property? Should that not be factored into the `isArray(val)` function check?